### PR TITLE
Prevent NaN

### PR DESCRIPTION
**Pull request by dlipeles**
Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave-intl/publishers/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Tagged reviewers.

Test Plan:

Reviewer Checklist:

Tests
- [ ] Adequate test coverage exists to prevent regressions

Security:
- [ ] No raw SQL -- Always prefer ActiveRecord query helpers ([more info on StackOverflow](https://stackoverflow.com/questions/41410752/rails-5-sql-injection#41452695))
- [ ] XSS is mitigated -- Avoid `html_safe` and `raw`; escape untrusted content from users and 3rd party APIs ([Rails XSS guide](https://brakemanpro.com/2017/09/08/cross-site-scripting-in-rails) and [OWASP XSS Prevention Cheat Sheet](https://www.owasp.org/index.php/XSS_(Cross_Site_Scripting)_Prevention_Cheat_Sheet))

### CODE REVIEW

File: app/javascript/packs/brave_rewards_banner_display.jsx [hunk 1]
```diff
@@ -50,10 +50,15 @@ class BraveRewardsPageForm extends React.Component {
...
   handleDonationAmountsChange(event) {
-    this.setState({
-      donationAmounts:
-        document.getElementById("donation-amounts-input").value.split(',').map(Number)
-    });
+     let donationAmounts = document.getElementById("donation-amounts-input").value.split(',').map(Number)
+
+     donationAmounts.forEach(function(amount){
+       if(isNaN(amount)){
+         donationAmounts = [1, 5, 10]
```
Comment (yachtcaptain23) [inline app/javascript/packs/brave_rewards_banner_display.jsx]: From a UX perspective, this is weird. I think the easier solution is to support strings or not verifying until the user is done typing or verifying upon submit.

**Review by yachtcaptain23 — CHANGES_REQUESTED** [review]
Comments in-line

**Comment by yachtcaptain23** [comment]
I'm seeing this, which shouldn't happen: 

image

**Comment by dlipeles** [comment]
I see why that's happening, the way brave-ui implemented the component it was never meant to have two buttons with the same token value. They are using token value as an identifier, and react breaks if there are two id's (token values in this case) that are the same. 

This presents a challenge because we need uniquely identify two elements that are the same, I found one way to do this, but it present more difficulty, will try to keep this code as clean as possible.

**Comment by yachtcaptain23** [comment]
No longer needed

**Pull request closed without merging**